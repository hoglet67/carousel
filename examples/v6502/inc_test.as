
        ;; Simple test ROM
        .ORG $8000
RSTVECTOR:      
        LDY #1
        LDA #1
        STA $00

TOP:
        INC $00
        LDA $00
        STA $00,Y
        INY

        JMP TOP

        ;; Reset vector      
        .ORG $FFFC
        .WORD RSTVECTOR
