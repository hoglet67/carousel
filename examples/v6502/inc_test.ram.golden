  
0000: 0e 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e ff ff   ................
0010: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0020: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0030: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0040: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0050: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0060: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0070: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
